<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chartWidget</class>
 <widget class="QMainWindow" name="chartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1041</width>
      <height>721</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>931</width>
        <height>241</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Brand</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Model</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Chipset</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>GPU</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Storage</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ram</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Price</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Year</string>
       </property>
      </column>
     </widget>
     <widget class="QSpinBox" name="sbPage">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>480</y>
        <width>441</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Filter</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>31</y>
         <width>421</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="3" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Year</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbS">
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>32</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>64</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>128</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>256</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>512</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1024</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Storage (GB)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="cbY">
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2017</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2018</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2019</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2020</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2021</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2022</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2023</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2024</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Ram</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="cbR">
          <item>
           <property name="text">
            <string>All</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>16</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>24</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>341</width>
        <height>263</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>ID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtID_2">
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dLabel_5">
         <property name="text">
          <string>Brand</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtBrand_2"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="txtModel_2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Chipset</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="txtChipset_2"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="tx_3">
         <property name="text">
          <string>GPU</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="txtGPU_2"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Storage</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="txtStorage_2"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>Ram</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="txtRam_2"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="dLabel_6">
         <property name="text">
          <string>Price</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="txtPrice_2"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="dQWDLabel_3">
         <property name="text">
          <string>Year</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="txtYear_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_Visual">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>390</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Visualization</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>931</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Search</string>
      </property>
      <widget class="QLineEdit" name="txtSearch_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>781</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbSearch_2">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>30</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Search</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Model</string>
        </property>
       </item>
      </widget>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>50</y>
         <width>120</width>
         <height>80</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="txt_total">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>80</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Total items:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>82</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Page:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnFilter">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>510</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Filter</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>360</y>
        <width>441</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Task</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>421</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="3" column="0">
         <widget class="QPushButton" name="btnSave">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="cbSort_2">
          <item>
           <property name="text">
            <string>Select arrangement</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sort by Model -  A --&gt; Z</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sort by Model - Z --&gt; A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sort by Price - Ascending</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sort by Price - Descending</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnDelete_2">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Sort:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="btnClear">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnAdd_2">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btnUpdate_2">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="1" column="0">
        <widget class="QPushButton" name="btnPrePage_2">
         <property name="text">
          <string>Previous Page</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnNextP_2">
         <property name="text">
          <string>Next Page</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1031</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
